### PR TITLE
largest-cake-pr

### PR DESCRIPTION
**Pull request by madhvi02**
@rish-singhal  Sir please have a look over this pr with all the suggested changes u asked for.
Issue: #1717

### CODE REVIEW

File: Graph/readme.md [hunk 1]
```diff
@@ -40,5 +40,6 @@ A graph is a pair (V, E), where V is a set of nodes, called vertices, and £ is
 - Depth First Search ----> [C++](/Code/C++/DFS.cpp) | [Java]() | [Python]()
 - Dijkstra Algorithm ----> [C++](/Code/C++/Dijkstra.cpp) | [Java]() | [Python]()
 - Kruskal Algorithm ----> [C++](/Code/C++/Kruskal.cpp) | [Java]() | [Python]()
+- Largest Piece of Cake ----> [C++](/Code/C++/largest_cake.cpp) | [Java]() | [Python]()
 - Topological Sort ----> [C++](/Code/C++/Topological_sort.cpp) | [Java]() | [Python]()
 - Negative Cycle In weighted Graph (Using Bellman–Ford algorithm) ---> [C++](/Code/C++/negative_cycle_in_weighted_graph.cpp) | [Java]() | [Python]()
```
Comment (rish-singhal) [inline Graph/readme.md:45]: Sort this list in alphabetical order

File: Code/C++/largest_cake.cpp [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+    {
+        vector<int>v;
+        cake.push_back(v);
+    }
+    for (int i = 0; i < n; ++i) {
+           for (int j = 0; j < n; ++j) {
+               cin >> cake[i][j];
+           }
+       }
+    cout<<getBiggestPieceSize(cake, n)<<endl;;
```
Comment (rish-singhal) [inline Code/C++/largest_cake.cpp:142]: Give spaces around <<, >>, …

File: Code/C++/largest_cake.cpp [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+    int ans=0;
+    visited[i][j]=1;
+    
+    //just checking in all the 4 directions and adding their respective ans into our main ans
+    //since the piece of the cake can be of any shape but it should be continuous,i.e the
+    //connected component of 1's will give you the length of a  piece of cake ;
+   
+    //top
+    int small_ans=helper(cake,visited,i-1,j,n);
+    ans+=small_ans;
```
Comment (rish-singhal) [inline Code/C++/largest_cake.cpp:83]: Give spaces around =,…

**Review by rish-singhal — CHANGES_REQUESTED** [review]

File: Graph/readme.md (hunk 1 above)
Comment (madhvi02) [reply Graph/readme.md:45]: Sir, I have placed mine. code's name in alphabetic order only .

**Pull request closed without merging**